docs/Migrating.md

```diff
@@ -13,10 +19,13 @@
      in the `Brain` GameObjects
   *  Agents have a `Brain` field in the Inspector, you need to drag the
   appropriate Brain asset in it.
-  
-__Note:__ You can pass the same brain to multiple agents in a scene by 
-leveraging Unity's prefab system or look for all the agents in a scene
-using the search bar of the `Hierarchy` window with the word `Agent`.
```

Comment:
The steps to do migration is still missing the steps to add new brain to the `BroadCast Hub`, and drag the brain into the corresponding field.

Reply:
Also the steps to drag the .tf file into the LearningBrain asset is missing. 

The steps to migrate from TensorFlowSharp to the Inference Engine are missing.

Reply:
Also add a note that all of the models need to be retrained, and the previous models no longer works.

Reply:
Also add a note that the model scope has been changed, and now each of the brain's checkpoint file lives within its own folder.